src/overlays/actors/ovl_En_Zov/z_en_zov.c

```diff
@@ -37,63 +40,593 @@ const ActorInit En_Zov_InitVars = {
+    EnZov* this = THIS;
+
+    Actor_MoveWithGravity(&this->actor);
+    Collider_UpdateCylinder(&this->actor, &this->collider);
+    CollisionCheck_SetOC(globalCtx, &globalCtx->colChkCtx, &this->collider.base);
+    Actor_UpdateBgCheckInfo(globalCtx, &this->actor, 10.0f, 10.0f, 10.0f, 4);
+
+    this->actionFunc(this, globalCtx);
+
+    if (!func_800EE29C(globalCtx, 0x1F8)) {
+        this->unk_320 &= ~0x10;
+    }
+
+    if ((this->unk_320 & 1) && func_80BD15A4(this, globalCtx)) {
+        func_800E9250(globalCtx, &this->actor, &this->unk_2F0, &this->unk_2F6, this->actor.focus.pos);
+    } else {
+        if ((this->unk_320 & 0x10) && (this->unk_322 == 0)) {
+            Math_SmoothStepToS(&this->unk_2F0.x, -0x1B58, 6, 0x1838, 0x64);
+        } else {
+            Math_SmoothStepToS(&this->unk_2F0.x, 0, 6, 0x1838, 0x64);
+        }
+        Math_SmoothStepToS(&this->unk_2F0.y, 0, 6, 0x1838, 0x64);
+        Math_SmoothStepToS(&this->unk_2F6.x, 0, 6, 0x1838, 0x64);
+        Math_SmoothStepToS(&this->unk_2F6.y, 0, 6, 0x1838, 0x64);
+    }
+
+    if (DECR(this->unk_2EE) == 0) {
+        this->unk_2EE = Rand_S16Offset(0x3C, 0x3C);
+    }
+
+    this->unk_2EC = this->unk_2EE;
+    if (this->unk_2EC >= 3) {
+        this->unk_2EC = 0;
+    }
+
+    if ((this->unk_32C != -1) && (ActorCutscene_GetCurrentIndex() != this->unk_328[this->unk_32C])) {
+        if ((this->unk_32C == 0) && (ActorCutscene_GetCurrentIndex() == 0x7C)) {
+            ActorCutscene_Stop(0x7C);
+            ActorCutscene_SetIntentToPlay(this->unk_328[this->unk_32C]);
+        } else if (ActorCutscene_GetCanPlayNext(this->unk_328[this->unk_32C])) {
+            ActorCutscene_Start(this->unk_328[this->unk_32C], &this->actor);
+        } else {
+            ActorCutscene_SetIntentToPlay(this->unk_328[this->unk_32C]);
+        }
+    }
+}
+
+s32 EnZov_OverrideLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot, Actor* thisx) {
+    EnZov* this = THIS;
+
+    if (limbIndex == 12) {
+        rot->x += this->unk_2F0.y;
+        if ((this->unk_320 & 0x10) && (this->unk_322 == 0)) {
+            rot->z += this->unk_2F0.x;
+        }
+    }
+
+    if (limbIndex == 11) {
+        rot->x += this->unk_2F6.y;
+    }
+    return false;
+}
+
+void EnZov_PostLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3s* rot, Actor* thisx) {
+    EnZov* this = THIS;
+
+    if (limbIndex == 12) {
+        Matrix_MultiplyVector3fByState(&D_80BD2744, &this->actor.focus.pos);
+        Math_Vec3f_Copy(&this->unk_2FC, &this->actor.focus.pos);
+        this->unk_2FC.y += 10.0f;
+    }
+
+    if (limbIndex == 18) {
+        Matrix_MultiplyVector3fByState(&D_80BD2750, &this->unk_308);
+    }
+
+    if (limbIndex == 13) {
+        Matrix_MultiplyVector3fByState(&D_80BD2750, &this->unk_314);
+    }
+}
+
+void EnZov_Draw(Actor* thisx, GlobalContext* globalCtx) {
+    EnZov* this = THIS;
+    Gfx* gfx;
+    s32 curFrame;
+    s32 phi_a1;
+    u8 phi_v1;
+
+    if (0) {}
```

Comment:
```suggestion
    if (1) {}
```
this is preferred over `if (0) {}` if it still matches

Reply:
I thought 0 was preferred over 1, because it's even less intrusive?

Reply:
In what sense do you mean intrusive? My understanding was that `if (1) {}` was preferred just for consistency's sake

Reply:
I mean, `if (1) {}` is essentially trying to say "always include this." Whereas `if (0) {}` is saying "never include this." So if(0) seems preferable if possible. Someone's told me to do exactly this too btw, but I don't remember who. Either Anghelo or Elliptic. I don't really mind either way, but I was asked to change it before and that was the reasoning as far as I remember.

Reply:
We usually use 1 unless there's reason to use 0.